src/runtime-rs/crates/runtimes/common/src/error.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use crate::types::{ProcessID, Response};
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error("failed to find container {0}")]
+    ContainerNotFound(String),
+    #[error("failed to find process {0}")]
+    ProcessNotFound(ProcessID),
+    #[error("failed to trans from response {0}")]
```

Comment:
I'd expand this as it's not clear what "trans" means here - translate, transfer, transliterate, transaction, transpose, ..., transcendentalization, transputer, transom, transmutation, _etc_, ... ? :smile:

Reply:
Thanks for your review, it's trying to express the structure translation to shim define failed, already updated.